dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/artifactbrowser/SendItemRequestAction.java

```diff
@@ -118,7 +118,11 @@ public Map act(Redirector redirector, SourceResolver resolver, Map objectModel,
                 )
                 .getRequestItemAuthor(context, item);
 
-        RequestItem requestItem = new RequestItem(item.getID(), Integer.parseInt(bitstreamId), requesterEmail, requesterName, message, Boolean.getBoolean(allFiles));
+	// probably wrong -- Boolean.getBoolean() returns true if and only if
+	// the system property named by the argument exists and is equal to the
+	// string "true".
+        // RequestItem requestItem = new RequestItem(item.getID(), Integer.parseInt(bitstreamId), requesterEmail, requesterName, message, Boolean.getBoolean(allFiles));
```

Comment:
Hi @wilee53 : You actually still can update this PR (based on this feedback).  All you need to do is:

1. Checkout your "DS-3294-new" branch (`git checkout DS-3294-new`)
2. Make the changes
3. Commit them (`git commit -a`)
4. Finally do a `git push origin DS-3294-new`.  This last statement will push your changes up to your GitHub and *automatically updates this PR*